bittensor/config.py

```diff
@@ -5,6 +5,7 @@
 
 import bittensor
 from bittensor.crypto import Crypto
+from bittensor.exceptions.ResponseExceptions import RemoteIPException
```

Comment:
@shibshib @parall4x  since we only have a few exceptions and they extend beyond "Response Exceptions" should we just rename ResponseExceptions to Exceptions?

Reply:
Done.